drivers/ethernet/eth_stm32_hal_v2.c

```diff
@@ -45,84 +23,18 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 #include "eth.h"
 #include "eth_stm32_hal_priv.h"
 
-#if DT_INST_PROP(0, zephyr_random_mac_address)
-#define ETH_STM32_RANDOM_MAC
-#endif
-
-#if defined(CONFIG_ETH_STM32_HAL_USE_DTCM_FOR_DMA_BUFFER) && \
-	    !DT_NODE_HAS_STATUS_OKAY(DT_CHOSEN(zephyr_dtcm))
-#error DTCM for DMA buffer is activated but zephyr,dtcm is not present in dts
-#endif
-
-static const struct device *eth_stm32_phy_dev = DEVICE_DT_GET(DT_INST_PHANDLE(0, phy_handle));
-
-#if DT_HAS_COMPAT_STATUS_OKAY(st_stm32h7_ethernet)
-#define IS_ETH_DMATXDESC_OWN(dma_tx_desc)	(dma_tx_desc->DESC3 & \
-							ETH_DMATXNDESCRF_OWN)
-
-#define ETH_RXBUFNB	ETH_RX_DESC_CNT
-#define ETH_TXBUFNB	ETH_TX_DESC_CNT
-
-/* Only one tx_buffer is sufficient to pass only 1 dma_buffer */
-#define ETH_TXBUF_DEF_NB	1U
-#else
-
-#define IS_ETH_DMATXDESC_OWN(dma_tx_desc)	(dma_tx_desc->Status & \
-							ETH_DMATXDESC_OWN)
-
-#endif /* DT_HAS_COMPAT_STATUS_OKAY(st_stm32h7_ethernet) */
+#define LOG_MODULE_NAME eth_stm32_hal_v2
```

Comment:
shouldn't the stm32 eth driver share one module name?

Reply:
Done